src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathfilter/InputSuppressionXpathFilterEscape.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suppressions PUBLIC
+        "-//Checkstyle//DTD SuppressionXpathFilter Experimental Configuration 1.2//EN"
+        "https://checkstyle.org/dtds/suppressions_1_2_xpath_experimental.dtd">
+<suppressions>
+    <suppress-xpath
+            files="Test"
+            id="777"
+            query="/CLASS_DEF[./IDENT[@text='InputSuppressionXpathFilterEscape']]/OBJBLOCK/VARIABLE_DEF[./IDENT[@text='quoteChar']]/ASSIGN/EXPR/PLUS[./STRING_LITERAL[@text='\&quot;escaped\&quot;']]"/>
```

Comment:
please wrap this long lines like it is done in 
https://github.com/checkstyle/checkstyle/blob/master/config/suppressions-xpath.xml#L9

Reply:
Thanks for the reference. On it.

Reply:
Done